File: javascript/module2/solutions/solution1.js
function getTotalFuelCostForTravel(startPoint, endPoint) {
  const fuelPerMeter = 6;

  const distance = Math.sqrt(
    Math.pow(endPoint.x - startPoint.x, 2) +
      Math.pow(endPoint.y - startPoint.y, 2)
  );

  const requiredFuel = distance * fuelPerMeter;

  return getTotalFuelCost(requiredFuel);
}

File: csharp/module2/solutions/solution1.cs
public class Point
{
    public double X { get; set; }
    public double Y { get; set; }
}

public class TravelCalculator
{
    private const double FuelPerMeter = 6;

    public double GetTotalFuelCostForTravel(Point startPoint, Point endPoint)
    {
        double distance = Math.Sqrt(
            Math.Pow(endPoint.X - startPoint.X, 2) +
            Math.Pow(endPoint.Y - startPoint.Y, 2)
        );

        double requiredFuel = distance * FuelPerMeter;

        return GetTotalFuelCost(requiredFuel);
    }
}
